Vindicta.Altis/Templates/Factions/WW2_CIVILIAN.sqf

```diff
@@ -0,0 +1,144 @@
+/*
+Civilian template for ARMA III
+*/
+
+// //==== Infantry ====
+_inf = [];
+_inf resize T_INF_SIZE;
+ _inf set [T_INF_DEFAULT, ["I_L_Looter_SG_F"]];				//Default infantry if nothing is found
+for "_i" from 1 to (T_INF_SIZE-1) do {
+    _inf set [_i, []];
+};
+// _inf set [T_INF_SL, ["B_Captain_Dwarden_F"]];
+// _inf set [T_INF_TL, ["B_Captain_Dwarden_F"]];
+// _inf set [T_INF_officer, ["B_GEN_Commander_F"]];
+// _inf set [T_INF_GL, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_rifleman, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_marksman, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_sniper, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_spotter, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_exp, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_ammo, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_LAT, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_AT, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_AA, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_LMG, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_HMG, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_medic, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_engineer, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_crew, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_crew_heli, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_pilot, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_pilot_heli, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_survivor, ["B_GEN_Soldier_F"]];
+// _inf set [T_INF_unarmed, ["B_GEN_Soldier_F"]];
+
+//==== Vehicles ====
+_veh = +(tDefault select T_VEH);
+_veh set [T_VEH_SIZE-1, nil];
+_veh set [T_VEH_default, [
+    "LIB_GazM1",
+    "LIB_GazM1_dirty",
+    "LIB_FRA_CitC4",
+    "LIB_FRA_CitC4Ferme"
+]];
+_vehWeights = [];
+_vehWeights set [T_VEH_SIZE-1, nil];
+_vehWeights set [T_VEH_default, [
```

Comment:
You need to update this bit as well, it represents the chance of spawning for each type. It should be same length array as T_VEH_default above.

Reply:
on it

Reply:
done